src/igvfd/tests/test_upgrade_in_vitro_system.py

```diff
@@ -38,11 +38,18 @@ def test_in_vitro_system_upgrade_5_6(upgrader, in_vitro_system_v5):
 def test_in_vitro_system_upgrade_6_7(upgrader, in_vitro_system_v6):
     value = upgrader.upgrade('in_vitro_system', in_vitro_system_v6, current_version='6', target_version='7')
     assert value['schema_version'] == '7'
-    assert value['taxa'] != 'Saccharomyces'
+    assert value['taxa'] == 'Saccharomyces'
```

Comment:
Should this be removed like the rest of the upgrades?

Reply:
also removed taxa here, thanks for pointing that out